e2e/dractions/actions.go

```diff
@@ -79,9 +80,11 @@ func EnableProtection(ctx types.Context) error {
 		return err
 	}
 
-	// For volsync based replication we must create the cluster namespaces with special annotation.
-	if err := util.CreateNamespaceAndAddAnnotation(ctx.Env(), ctx.AppNamespace(), log); err != nil {
-		return err
+	// For Kubernetes, creates namespaces and adds annotation on both DR clusters for volsync based replication.
+	if config.Distro == e2econfig.DistroK8s {
```

Comment:
To avoid conflicts with the package name we can use shorter name for the config like cfg.